.github/workflows/publish_to_hex_pm.yml

```
@@ -11,8 +11,8 @@ on:
   workflow_dispatch:
 
 env:
-  OTP_VERSION: 24.0.4
-  ELIXIR_VERSION: 1.12.2
+  OTP_VERSION: 24.2.1
```

they just released a new version 😆
```suggestion
  OTP_VERSION: 24.2.2
```

---

Thanks @longnd I upgrade it in https://github.com/nimblehq/elixir-templates/pull/169/commits/3bcba4a8cd51a87c28d5b4465904edb6078ddb09